config/racecar.rb

```diff
@@ -3,9 +3,9 @@
   config.group_id_prefix = 'compliance'
 
   if Settings.kafka.security_protocol
-    config.security_protocol = Settings.kafka.security_protocol
+    config.security_protocol = Settings.kafka.security_protocol.to_sym
 
-    if config.security_protocol == 'sasl_ssl'
+    if config.security_protocol == :sasl_ssl
```

Comment:
Oneline Alternative: 
```
if Settings.kafka.security_protocol == 'sasl_ssl'
```
Clowder values are usually strings, unless our Settings would cast them (in future).